### PR TITLE
Announcement structure

### PR DESCRIPTION
**Pull request by Vidmapps**
Created this as a branch for the previous branch by mistake, I guess this messed up things? Will I be able to merge this one to master?

### CODE REVIEW

**Comment by Vidmapps** [comment]
Oh, fixed some HTML and it seems I will be able 😅

**Comment by alynioke** [comment]
ye, looks good.
again - doing visual changes (like adding smth new), please add screenshot for reviewer ;)

**Comment by Vidmapps** [comment]
Desktop 
image

Looks HOT for me :D

**Comment by Vidmapps** [comment]
image

File: index.html [hunk 1]
```diff
@@ -218,8 +218,18 @@
...
-                            <div class="flex-and-space-between dashboard-name">blog</div>
+                        <div class="announcement border-radius tab-structure">
+                            <div class="announcement-name uppercasing announcement-padding">From our blog</div>
+                            <div class="announcement-header announcement-padding">Urgent ! We Updated Our ToS</div>
+                            <div class="announcement-content flex-and-space-between">
+                                <div class="announcement-content-text">
+                                    <div class="announcement-text announcement-padding">Hello, we just updated our tos..</div>
+                                    <div class="announcement-read-more uppercasing announcement-padding">Read more</div>
+                                </div>
+                                <div class="announcment-image">
```
Comment (alynioke) [inline index.html:229]: here and in few other lines it's `announcment` and should be announcEment

File: index.html [hunk 2]
```diff
@@ -218,8 +218,18 @@
...
+                                    <img src="img/img.svg">
```
Comment (alynioke) [inline index.html:230]: i recommend naming all images according what they are displaying. not just "img" but "tos" or "terms-of-service" etc

**Comment by alynioke** [comment]
> Looks HOT for me :D

Yass!
image

File: index.html (hunk 1 above)
Comment (Vidmapps) [reply index.html]: I was so used to spell it wrong.....

**Pull request merged**